packages/tables/resources/lang/pl/table.php

```diff
@@ -107,7 +125,7 @@
 
     'selection_indicator' => [
 
-        'selected_count' => '{1} 1 rekord zaznaczony.|[2,4]:count rekordy zaznaczone.|[5,*]:count rekordów zaznaczonych.',
```

Comment:
I would keep the previous version. It is grammatically correct, e.g. for 3 pieces. And the user has value in one place which is more intuitive, and there's no need to look for values. 👀 
X rekord zaznaczony.
Zaznaczono rekordów: X

Reply:
Yes, it is correct and I'd rather keep it, but it doesn't appear to be working.
- 1 rekord zaznaczony.
- 2 rekordy zaznaczone.
- 4 rekordy zaznaczone.
- 5 rekordy zaznaczone (!)

Reply:
The problem is not in the translation, but in another place. You can run `tinker`, and paste:
`Lang::choice('{1} 1 rekord zaznaczony.|[2,4]:count rekordy zaznaczone.|[5,*]:count rekordów zaznaczonych.', 5)`
And it works well.

Reply:
I see, so something in tables? Picking your version, then.